src/_PlotRecipes.jl

```diff
@@ -1,7 +1,7 @@
 # B-spline space
 @recipe function f(P::AbstractFunctionSpace)
     # TODO fix number of sampling points
-    N = 100
+    N = 50
```

Comment:
Reducing the number of sampling points does not reduce the loading time of the documentation, so I will revert the value.
```suggestion
    N = 100
```